frontend/src/actionCreators/varianceActionCreator.js

```diff
@@ -60,6 +60,20 @@ export const fetchVariancesByMine = ({ mineGuid }) => (dispatch) => {
     .finally(() => dispatch(hideLoading()));
 };
 
+export const fetchVariancesById = (mineGuid, varianceGuid) => (dispatch) => {
+  dispatch(request(reducerTypes.GET_VARIANCE));
+  dispatch(showLoading("modal"));
+  return CustomAxios(Strings.ERROR)
+    .get(ENVIRONMENT.apiUrl + API.VARIANCE(mineGuid, varianceGuid), createRequestHeader())
+    .then((response) => {
+      dispatch(success(reducerTypes.GET_VARIANCE));
+      dispatch(varianceActions.storeVariance(response.data));
+      dispatch(hideLoading("modal"));
```

Comment:
do we need to dispatch hide loading modal here if the 'finally' block does it?

Reply:
good catch